### PR TITLE
doc: Add features of nRF70 Series documentation

### PR DESCRIPTION
**Pull request by divipillai**
Added documentation for features of nRF70 Series

Signed-off-by: divya pillai <divya.pillai@nordicsemi.no>

### CODE REVIEW

**Review by richabp — APPROVED** [review]

File: doc/nrf/ug_nrf70_features.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+* :ref:`nrf7002dk_nrf5340`
+* The :ref:`introductory documentation <getting_started>` if you are not familiar with the |NCS| and the development environment.
+
+The following figure illustrates the partitioning of the Wi-Fi stack between the nRF70 series device and the host device:
+
+.. figure:: images/nrf70_ug_overview.svg
+   :alt: Overview of nRF71 application architecture
+
+   Overview of nRF71 application architecture
```
Comment (bama-nordic) [inline doc/nrf/ug_nrf70_features.rst:29]: Change nRF71 to nRF70

**Review by bama-nordic — CHANGES_REQUESTED** [review]

**Review by bama-nordic — APPROVED** [review]
LGTM

File: doc/nrf/ug_nrf70_features.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+.. contents::
+    :local:
+    :depth: 2
+
+Introduction
+************
+
+The nRF70 Series of wireless companion ICs add Wi-Fi capabilities to another System-on-Chip (SoC), Microprocessor Unit (MPU), or Microcontroller (MCU) host device.
+It implements the Physical (PHY) and Medium Access Controller (MAC) layers of the 802.11 protocol stack, while the higher layers of the networking stack run on the host device.
```
Comment (mjd90) [inline doc/nrf/ug_nrf70_features.rst:14]: 802.11 => IEEE802.11

File: doc/nrf/ug_nrf70_features.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+   * - nRF7002 DK
+     - PCA10143
+     - ``nrf7002dk_nrf5340_cpuapp``
+     -
+
+Supported Wi-Fi standards and modes
+===================================
+
+The nRF70 series wireless companion ICs add Wi-Fi 6 support to a host device that includes IP-based networking support.
+Wi-Fi 6 aligns with IEEE802.11ax and all earlier versions of the 802.11 suite of wireless LAN standards.
```
Comment (mjd90) [inline doc/nrf/ug_nrf70_features.rst:52]: 802.11 => IEEE802.11

File: doc/nrf/ug_nrf70_features.rst (hunk 2 above)
Comment (divipillai) [reply doc/nrf/ug_nrf70_features.rst:14]: As this PR is merged, these two suggestions will be fixed in https://github.com/nrfconnect/sdk-nrf/pull/9905/files.

Comment (richabp) [reply doc/nrf/ug_nrf70_features.rst:14]: Fixed in https://github.com/nrfconnect/sdk-nrf/pull/9905

File: doc/nrf/ug_nrf70_features.rst (hunk 3 above)
Comment (richabp) [reply doc/nrf/ug_nrf70_features.rst:52]: Fixed in https://github.com/nrfconnect/sdk-nrf/pull/9905

**Pull request merged**